### PR TITLE
feat: handle partial withdraw. Add info about staking pool expiration.

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: src/env.js [hunk 1]
```diff
@@ -13,6 +13,6 @@
     window.__env.cacheServerUrl = 'https://identitycache-dev.energyweb.org/v1';
     window.__env.natsServerUrl = 'https://identityevents-dev.energyweb.org/';
     window.__env.kmsServerUrl = undefined;
-    window.__env.stakingPoolFactoryAddress = '0x3490B3e5E7C1f696c61FDe1c2679725B19274B0e';
+    window.__env.stakingPoolFactoryAddress = '0xd61Ac65c0fECfb83108f6E374a57FfF9c692901d';
```
Comment (Malak67) [inline src/env.js:16]: Shouldn`t this stuff be taken from a .env file and not commited? Only if it`s sensitive data. If not, they maybe it`s ok

File: src/app/routes/ewt-patron/withdraw/withdraw.component.html [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
-    </div>
+<div class="d-flex justify-content-center align-items-center flex-column stake-success-container">
+  <h3 class="mt-3 mb-4">Withdraw?</h3>
+  <p class="text-center mt-1">
+    EWT will be transfered to your account balance.
+    You will be charged with gass fee.
+  </p>
+</div>
+
+<mat-form-field class="staking-textfield" appearance="outline">
```
Comment (Malak67) [inline src/app/routes/ewt-patron/withdraw/withdraw.component.html:9]: Usually it is a good approach to keep all form fields inside a form, even if it's only one.

File: src/app/shared/services/pool/staking-pool-facade.ts [hunk 3]
```diff
@@ -21,11 +21,21 @@ export class StakingPoolFacade {
...
+  getStartDate() {
+    return from(this.pool.getStart());
+  }
+
+  getEndDate() {
+    return from(this.pool.getEnd());
+  }
+
   checkReward() {
+    console.log('check reward');
```
Comment (Malak67) [inline src/app/shared/services/pool/staking-pool-facade.ts:33]: Is this intentional or missed here? Better to remove console.logs before merging but maybe you need to check something after it is deployed?

File: src/app/shared/services/pool/staking-pool-facade.ts [hunk 4]
```diff
@@ -21,11 +21,21 @@ export class StakingPoolFacade {
...
     return from(this.pool.checkReward());
   }
 
   getStake() {
+    console.log('get stake');
```
Comment (Malak67) [inline src/app/shared/services/pool/staking-pool-facade.ts:38]: Same question as above.

File: src/app/state/pool/pool.selectors.ts [hunk 5]
```diff
@@ -10,7 +10,7 @@ export const getStakeState = createFeatureSelector<PoolState>(USER_FEATURE_KEY);
 
 export const getReward = createSelector(
   getStakeState,
-  (state: PoolState) => state.reward
+  (state: PoolState) => formatEther(state.reward as any)
```
Comment (Malak67) [inline src/app/state/pool/pool.selectors.ts:13]: Why cast it as any? Can't you create a type or interface for it?

File: src/env.js (hunk 1 above)
Comment (dawidgil) [reply src/env.js:16]: It should, but right now devops do not take care of it. So it is maintained by repo.

File: src/app/shared/services/pool/staking-pool-facade.ts (hunk 3 above)
Comment (dawidgil) [reply src/app/shared/services/pool/staking-pool-facade.ts:33]: missed, thanks for catching this!

**Review by Malak67 — APPROVED** [review]

**Pull request merged**